ratis-docs/content/cli.md

```diff
@@ -41,12 +42,24 @@ $ mvn -DskipTests -Prelease -Papache-release clean package assembly:single
 
 Extract the ratis-shell tarball.
 ```
-$ tar -C <DST_DIR>/ratis-shell -xzf ratis-assembly/target/apache-ratis-<VERSION>-shell.tar.gz
+$ mkdir <DST_DIR>/ratis-shell
+$ tar -C <DST_DIR>/ratis-shell -xzf ratis-assembly/target/apache-ratis-<VERSION>-shell.tar.gz --strip-component 1
+```
+
+### Setting up from a binary tarball
+Download the Ratis bin tarball from https://ratis.apache.org/downloads.html .
+Note that ratis-shell is available starting from version 2.3.0 (already released) does not have ratis-shell bin.
```

Comment:
This sentence "... available ... does not have ..." is a little bit confusing.   How about we rewrite it as below?
```
Note that the bin tarball of Ratis version 2.3.0 or earlier does not contain ratis-shell.
The bin tarball of later versions will contain ratis-shell.
```